tifeatures/layer.py

```diff
@@ -261,6 +261,19 @@ def _datetime_filter_to_sql(self, interval: List[str], dt_name: str):
                     logic.V(dt_name) < logic.S(pg_funcs.cast(end, "timestamptz")),
                 )
 
+    def _sortby(self, sortby: Optional[str]):
```

Comment:
why not supporting multiple sortby directly?

Reply:
Only because I was trying to get something that would work for the base case out quickly!

Reply:
Last push adds support for multiple sortby.